Libraries/Lists/VirtualizedList.js

```diff
@@ -733,6 +752,175 @@ class VirtualizedList extends React.PureComponent<Props, State> {
     this.state = initialState;
   }
 
+  static _createRenderMask(
+    props: Props,
+    cellsAroundViewport: {first: number, last: number},
+    lastFocusedItem: ?number,
+  ): CellRenderMask {
+    const itemCount = props.getItemCount(props.data);
+
+    invariant(
+      cellsAroundViewport.first >= 0 &&
+        cellsAroundViewport.last >= cellsAroundViewport.first - 1 &&
+        cellsAroundViewport.last < itemCount,
+      `Invalid cells around viewport "[${cellsAroundViewport.first}, ${cellsAroundViewport.last}]" was passed to VirtualizedList._createRenderMask`,
+    );
+
+    const renderMask = new CellRenderMask(itemCount);
+
+    // Keep the items around the last focused rendered, to allow for keyboard
+    // navigation
+    if (lastFocusedItem) {
+      const first = Math.max(0, lastFocusedItem - 1);
+      const last = Math.min(itemCount - 1, lastFocusedItem + 1);
```

Comment:
Want to change this in the future to something more sophisticated, like keeping a single render batch's worth of content around instead of one on each side.